src/components/Login/PwdInputGroup/index.tsx

```diff
@@ -20,7 +21,7 @@ const PwdInputGroup: React.FC<Props> = ({ label, value, onChange }) => {
   const handleClick = () => setShow(!show);
 
   return (
-    <FormControl isRequired>
+    <FormControl isRequired style={{ marginTop: 8 }}>
```

Comment:
Why don't you use marginTop props? Any reason to use inline style here? If you can, please fix all shorten props in this component. Such as changing pr to paddingRight at line28.

Reply:
> Why don't you use marginTop props? Any reason to use inline style here? If you can, please fix all shorten props in this component. Such as changing pr to paddingRight at line28.

I've updated this component :)